tests/RobotFramework/tests/cumulocity/software_management/sm-plugin.robot

```diff
@@ -47,6 +47,14 @@ sm-plugins should work without sudo installed and running as root
     ${software}=    Device Should Have Installed Software    dummy1-0001,1.0.0::dummy1    dummy2-0001,1.0.0::dummy2
     Length Should Be    ${software}    2
 
+sm-plugins download files from Cumulocity
+    Execute Command    sudo tedge config set software.plugin.max_packages 1
+    Connect Mapper    c8y
+    Device Should Exist    ${DEVICE_SN}
+    ${file_url}=    Cumulocity.Create Inventory Binary    sm-plugin-test-file    software1    contents=Testing a thing
+    ${OPERATION}=    Install Software    dummy-software,1.0.0::dummy3,${file_url}
+    ${OPERATION}=    Operation Should Be SUCCESSFUL    ${OPERATION}    timeout=60
+    File Should Exist    /tmp/dummy3/intalled_dummy-software
```

Comment:
The test is correct but not really clear: the downloaded content is checked by the dummy plugin and not the test itself.

I suggest to add:

```
${downloaded}=    Execute Command    /tmp/dummy3/intalled_dummy-software
Should Be Equal    ${downloaded}    Testing a thing
```